### PR TITLE
[Owner Review]Added an example for NI DMM grpc APIs

### PR DESCRIPTION
**Pull request by ShantanuShinde**
### What does this Pull Request accomplish?

A C example. ContinuousAcquireMultiplePoints, has been ported to python using the NIDMM gRPC APIs

### Why should this Pull Request be merged?

This example can be used to test and understand the generated DMM APIs

### What testing has been done?

The example has been locally tested and is working as expected

### CODE REVIEW

**Review by harsha-bhushan — APPROVED** [review]

**Review by ganeshnie — APPROVED** [review]

File: examples/nidmm/continuous-acquire-graph-multiple-points.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
+# 
+# This example performs continuous multipoint acquisition and prints the number of points read and average value of measurements.
+#
+# The gRPC API is built from the C API.  NI-DMM documentation is installed with the driver at:
```
Comment (doshirohan) [inline examples/nidmm/continuous-acquire-graph-multiple-points.py]: nitpick: Looks like you have two spaces at the end of first statement here.

File: examples/nidmm/continuous-acquire-graph-multiple-points.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+powerlineFreq       = 60.0
+
+# Read in cmd args
+if len(sys.argv) >= 2:
+    server_address = sys.argv[1]
+if len(sys.argv) >= 3:
+    server_port = sys.argv[2]
+if len(sys.argv) >= 4:
+    resource = sys.argv[3]
+    options = ""
```
Comment (doshirohan) [inline examples/nidmm/continuous-acquire-graph-multiple-points.py:53]: Are options not required for non-simulation use case? If they are needed then we can expose them as parameter as well.

**Review by doshirohan — APPROVED** [review]

File: examples/nidmm/continuous-acquire-graph-multiple-points.py [hunk 3]
```diff
@@ -0,0 +1,198 @@
+# 
+# This example performs continuous multipoint acquisition and prints the number of points read and average value of measurements.
+#
+# The gRPC API is built from the C API.  NI-DMM documentation is installed with the driver at:
+# C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niDMM\Documentation\English\DMM.chm
+
```
Comment (jonlugoatx) [inline examples/nidmm/continuous-acquire-graph-multiple-points.py]: Remove blank line

File: examples/nidmm/continuous-acquire-graph-multiple-points.py [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+# Getting Started:
+#
+# To run this example, install "NI-DMM Driver" on the server machine.
+# Link : https://www.ni.com/en-us/support/downloads/drivers/download.ni-dmm.html
+#
+# For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC Client" wiki page.
+# Link: https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+#
+# Refer to the NI-DMM gRPC Wiki to determine the valid channel and resource names for your NI-DMM module.
+# Link : https://github.com/ni/grpc-device/wiki/NI-DMM-C-Function-Reference
```
Comment (jonlugoatx) [inline examples/nidmm/continuous-acquire-graph-multiple-points.py]: I don't think we'll have a DMM C-function reference wiki at the end of Q2. All that should be moving to ni.com. Work with Kevin Reimnitz to work out the details.

File: examples/nidmm/continuous-acquire-graph-multiple-points.py (hunk 2 above)
Comment (jonlugoatx) [reply examples/nidmm/continuous-acquire-graph-multiple-points.py:53]: I don't think it makes sense to add options as a separate command-line argument.

**Review by jonlugoatx — APPROVED** [review]

File: examples/nidmm/continuous-acquire-graph-multiple-points.py (hunk 3 above)
Comment (ShantanuShinde) [reply examples/nidmm/continuous-acquire-graph-multiple-points.py]: removed the space

File: examples/nidmm/continuous-acquire-graph-multiple-points.py (hunk 4 above)
Comment (ShantanuShinde) [reply examples/nidmm/continuous-acquire-graph-multiple-points.py]: removed that link, I will add the link to reference in later PR

File: examples/nidmm/continuous-acquire-graph-multiple-points.py (hunk 1 above)
Comment (ShantanuShinde) [reply examples/nidmm/continuous-acquire-graph-multiple-points.py]: removed extra space

File: examples/nidmm/continuous-acquire-graph-multiple-points.py (hunk 4 above)
Comment (ShantanuShinde) [reply examples/nidmm/continuous-acquire-graph-multiple-points.py]: I added the link to the ni.com page

File: examples/nidmm/continuous-acquire-graph-multiple-points.py [hunk 5]
```diff
@@ -12,6 +12,9 @@
 # For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC Client" wiki page.
 # Link: https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
 #
+# Refer to the NI DMM Support Manual page to determine the valid channel and resource names for your NI-DMM module.
+# Link : https://zone.ni.com/reference/en-XX/help/370384V-01/TOC151.htm
```
Comment (harsha-bhushan) [inline examples/nidmm/continuous-acquire-graph-multiple-points.py]: Remove this link. This is not the right link. All of ni's documentation would be moving to a new experience on ni.com (not the link you have pasted). However right now it is unlikely that the MI documentation would be moving by Q2. Hence we should go ahead with our plan of converting chm to mark down & submitting them in the wiki.
You can either retain that placeholder wiki link you had in your previous commit, or you could remove the link & add it once the documentation story is done (it is anyway captured as part of this task - https://dev.azure.com/ni/DevCentral/_workitems/edit/1420181)

Comment (ShantanuShinde) [reply examples/nidmm/continuous-acquire-graph-multiple-points.py]: removed the link

**Review by kyledsullivan — APPROVED** [review]

**Pull request merged**